src/python_testing/TC_MCORE_FS_1_5.py

```diff
@@ -0,0 +1,290 @@
+#
+#    Copyright (c) 2024 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+# TODO: add to CI. See https://github.com/project-chip/connectedhomeip/issues/34676
+# for details about the block below.
+#
```

Comment:
There is no block below. Does this need re-wording explaining why not in CI (maybe separate issue)?

Reply:
We are working on getting FS DUT working in CI but it won't be ready in time for SVE FC


I have since removed the "for details about the block below."